omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -218,4 +218,12 @@
 		}
 		return appList;
 	}
+
+	@RequestMapping(value = "/rest/owa/allowAdmin", method = RequestMethod.GET)
```

Comment:
What does allowAdmin mean? Does it mean allow admin to delete users? Or does it mean something else?

Reply:
I chose to use `allowAdmin` as the endpoint since the property doesn't just  restrict the Admin from not just deleting the app but all other settings the admin has on the webapp.

Reply:
allowAdmin is a bad name. It is too general. A good variable name should state what the admin is being allowed to.

Reply:
What do you think of `adminPriviledges`?

Reply:
@dkayiwa

Reply:
Something like allowModuleWebUpload

Reply:
thank you, let me update it.